app/views/evaluation_mailer/evaluation.html.erb

```diff
@@ -0,0 +1,17 @@
+<p>
+  Hej kära SOF-besökare,
+</p>
+
+<p>
+  Vi från organisationen vill tacka för ett lyckat SOF17 och hoppas att ni hade det lika roligt som vi hade!
+</p>
+
+<p>
+  För att nästa festival ska bli ännu bättre är det jätteviktigt att du tar dig en minut att <b>fylla i denna utvärdering:</b> <a href="https://goo.gl/forms/TMw0zeL0SFcKOBgn2">Klicka mig!</a>
+</p>
+
+<p>
+  Som tack för allt har vi genom vår samarbetspartner FKP Scorpio fått möjligheten att erbjuda dig <b>15% rabatt till Bråvallafestivalen</b> den 28 juni till 1 juli i Norrköping. Så packa festivalväskan och köp dina biljetter <a href="https://www.biljettforum.se/direkte/?token=ZBixHRJxbOcH4uNAa1MWDpLBxN%2bGpU3XuU8twpYQLXwBvbzV56Aoaj6CEWLM5ZL1%2fqosS7%2f9CKPbYHQJmfPWcQo8Uw6Q6jLk5syszj67JGR9p%2b%2b%2bNpzWE2LwwSs0GhxB">Här!</a>
+</p>
+
+<img src="https://s3-eu-west-1.amazonaws.com/lintek-sof/webapp/festival/studenterbjudande-mail.png">
```

Comment:
I think this image should have an inline style to fill the width, right now it may overflow the content area and require scrolling to show it all.